res/values/arrays.xml

```diff
@@ -20,11 +20,13 @@
   <string-array name="themeKeys" translatable="false">
     <item>themeLight</item>
     <item>themeDefault</item>
+    <item>themeSystem</item>
   </string-array>
 
   <string-array name="themes">
     <item>Light theme</item>
     <item>Dark theme (default)</item>
+    <item>System theme</item>
```

Comment:
Won't this cause issues for translations if the translated string-arrays only have 2 entries?
Or will it be clever enough to fall back to English for just the missing entry?

Reply:
Good catch. I tested this, and it crashes. I'll look at fixing it.